tests/lit-tests/loop_unroll_varying.ispc

```diff
@@ -0,0 +1,213 @@
+// Test to check '#pragma unroll' functionality for different loop statements
+//   with varying type induction variables.
+
+// RUN: %{ispc} %s --target=host --nowrap -O0 --emit-llvm-text --no-discard-value-names --nostdlib -o - | FileCheck %s -check-prefixes=CHECK_ALL,CHECKO0
+// RUN: %{ispc} %s --target=host --nowrap -O2 --emit-llvm-text --no-discard-value-names --nostdlib -o - | FileCheck %s -check-prefixes=CHECK_ALL,CHECKO2
+
+// XFAIL: LLVM_15_0+
```

Comment:
Why is it failing with LLVM 15?

Reply:
Not sure yet - investigating. I am assuming that the test is fragile due to some change in the code structure in the newer version.